code/src/java/pcgen/cdom/facet/FacetLibrary.java

```diff
@@ -33,12 +32,11 @@
 	
 	private FacetLibrary()
 	{
-		//Do not instantiate
 	}
 
 	private static Map<Class<?>, Object> facets = new HashMap<>();
 
-	public static <T extends Object> T getFacet(Class<T> cl)
+	public static <T> T getFacet(Class<T> cl)
```

Comment:
This is from an automated linter. I did not actually know about this subtlety with 'extends from Object'. 
I'll drop this.